Missing lodash dependency

lodash is not a dependency of unexpected-express, but a transitive dependency. Under normal circumstances it will work, as magicpen-media depends on lodash and is a dependency of unexpected-messy, which itself is a dep of this module. But if you have a dependency of lodash that is not compatible with magicpen-media, lodash will be hoisted into the magicpen-media/node_modules folder.

intersection is added to lodash in v3.0.0, so you need to use v2 of lodash to trigger this error.

The following works fine:

```
$ mkdir foobar && cd foobar
$ npm init -y
$ npm i -S unexpected-express
$ node node_modules/unexpected-express/lib/resolveExpectedResponseProperties.js 
$ echo $?
0
```

To reproduce the error, continue:

```
$ npm i lodash@^2
$ node node_modules/unexpected-express/lib/resolveExpectedResponseProperties.js 
module.js:550
    throw err;
    ^

Error: Cannot find module 'lodash/intersection'
[...]
```

--- a/package.json
+++ b/package.json
@@ -62,6 +62,7 @@
     "createerror": "^1.1.0",
     "form-data": "^3.0.0",
     "is-stream": "^2.0.0",
+    "lodash": "^4.0.0",
     "messy": "^6.6.1",
     "qs": "^6.5.1",
     "unexpected-messy": "^9.0.0"

